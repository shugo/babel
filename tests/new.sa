class MAIN is
   main is
      obj ::= new System::Object;
      #OUT + obj.ToString + "\n";
      ary ::= new System::Collections::ArrayList(10);
      #OUT + ary.ToString + "\n";
   end;
end;
