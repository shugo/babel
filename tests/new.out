System.Object
System.Collections.ArrayList
